src/EventTickets/resources/components/EventTicketsHeader.tsx

```diff
@@ -0,0 +1,16 @@
+import moment from 'moment';
+
+export default function EventTicketsHeader({title, date}) {
+    const fullDate = moment(date).format('dddd, MMMM Do, h:mma z');
+    const [day, month] = moment(date).format('DD MMM').split(' ');
```

Comment:
I would suggest `moment`'s more declarative methods, such as [moment().date()](https://momentjs.com/docs/#/get-set/date/) and [monent().month()](https://momentjs.com/docs/#/get-set/month/), instead of parsing a format string.

Reply:
Using those proposed methods I'd need to manually format the values. An alternative to eliminate the parsing could be like:
```js
const day = moment(date).format('DD');
const month = moment(date).format('MMM');
```